### PR TITLE
Profile wrapup

### LINKED ISSUE #172: add Profile Plot feature
This feature has been partially implemented.  Looking to finish it today. 

Current State covers basic profiles... still not great support for `Weir` type. 

## Simple
test

## Orifice
test_orifice

## Weir
test_weir

### PR DESCRIPTION
**Pull request by bemcdonnell**
Closes #172

### CODE REVIEW

**Comment by bemcdonnell** [comment]
test_weir
Here is what I ended up doing with Weirs.

File: swmmio/tests/data/SWMMExtCntrlModel.inp [hunk 1]
```diff
@@ -0,0 +1,2601 @@
...
+;;Node           Parameter        Time Series      Type     Factor   Factor   Value    Pattern
+;;-------------- ---------------- ---------------- -------- -------- -------- -------- --------
+J13              FLOW             10002_TimeSeries FLOW     1.0      1        0
+J18              FLOW             10002_TimeSeries FLOW     1.0      1        0
+J28              FLOW             10002_TimeSeries FLOW     1.0      1        0
+
+[TIMESERIES]
+;;Name           Date       Time       Value
+;;-------------- ---------- ---------- ----------
+10002_TimeSeries 10/01/2016 00:00      0.000356481487475
```
Comment (aerispaha) [inline swmmio/tests/data/SWMMExtCntrlModel.inp:190]: @bemcdonnell do we need this timeseries in the inp for the unit tests? if not, can we drop this, just to keep the file size down a bit?

Comment (bemcdonnell) [reply swmmio/tests/data/SWMMExtCntrlModel.inp:190]: @aerispaha probably not hugely required.  I could remove if you would like

Comment (aerispaha) [reply swmmio/tests/data/SWMMExtCntrlModel.inp:190]: @bemcdonnell awesome, thank you!

**Comment by bemcdonnell** [comment]
@aerispaha I force pushed the reduced lines... didn't know you were working on it :)

**Comment by aerispaha** [comment]
> pushed the reduced lines... didn't know you were working on it

awesome, thanks! love that commit message btw :)

**Comment by bemcdonnell** [comment]
@aerispaha Do you want to drop appveyor soon?

**Comment by aerispaha** [comment]
@bemcdonnell re: appveyor, yes I would like to drop appveyor soon. Just need to port over the deploy logic such that we can automatically post new versions to PyPI on tagged versions

**Comment by bemcdonnell** [comment]
let's raise that in a separate issue.

**Review by aerispaha — APPROVED** [review]
party on 🥳 
thanks for this contribution, @bemcdonnell !

**Comment by bemcdonnell** [comment]
@aerispaha happy to help!

**Comment by aerispaha** [comment]
> let's raise that in a separate issue.

Here it is! #175

**Pull request merged**